Loadouts/fin_ak103_m05s.hpp

```diff
@@ -126,8 +127,7 @@ class uav: rifleman {
 };
 class ar: rifleman {// AR
   weapons[] = {AR};
-  attachments[] = {};
-  opticChoices[] = {AR_ATTACHMENTS};
+  attachments[] = {AR_ATTACHMENTS};
```

Comment:
I'm fairly sure this is correct?
I didn't get why the bipod on the `choices[]`

Reply:
Seems correct to me as well - probably a bad Copy/Paste for attachments from somewhere else.